rs/cli/src/commands/completions.rs

```diff
@@ -11,7 +11,7 @@ pub struct Completions {
 
 impl ExecutableCommand for Completions {
     fn require_ic_admin(&self) -> super::IcAdminRequirement {
-        super::IcAdminRequirement::None
+        super::IcAdminRequirement::none()
```

Comment:
If you want to, you can name them and return via functions just please don't use `default`.